pytest/tests/sanity/catchup_flat_storage_deletions.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python3
+# Spins up a validator node tracking all shards and a non-validator node.
+# Deletes an account.
+# Restart the non-validator node.
+# After the non-validator node does catchup, attempt to send a token to the deleted account.
+# Observe that both nodes correctly execute the transaction and return an error.
```

Comment:
Is this observation verified in any way, or just something you could see in the printed logs?

Reply:
It's verified by the second node not getting stuck.
Added a direct check that `test_account_id` doesn't exist on the node.